packages/web-app/src/components/common/ExploredCaves/ExploredCavesList.jsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { useDispatch } from 'react-redux';
+import { Button, Tooltip, Divider, Typography } from '@mui/material';
+import AddCircleIcon from '@mui/icons-material/AddCircle';
+import CancelIcon from '@mui/icons-material/Cancel';
+import { styled } from '@mui/material/styles';
+
+import CavesList from '../cave/CavesList';
+import EntrancesList from '../entrance/EntrancesList';
+import Alert from '../Alert';
+import { NetworkPropTypes } from '../../../types/grotto.type';
+import { EntranceSimplePropTypes } from '../../../types/entrance.type';
+import SearchCaveForm from '../../appli/Form/SearchCaveForm';
+import { addCaveExplorer } from '../../../actions/Cave/AddExplorer';
+import { removeCaveExplorer } from '../../../actions/Cave/RemoveExplorer';
+
+const DividerStyled = styled(Divider)`
+  background-color: ${props => props.theme.palette.divider};
+`;
+
+const ExploredCavesList = ({
+  exploredEntrances,
+  exploredNetworks,
+  explorerId,
+  isOrganization = true,
+  canManage,
+  onRefresh
+}) => {
+  const { formatMessage } = useIntl();
+  const dispatch = useDispatch();
+  const [isCaveSearchVisible, setIsCaveSearchVisible] = useState(false);
+  const [isAdding, setIsAdding] = useState(false);
+
+  const fetchCaveIdFromEntrance = async entranceId => {
+    const response = await fetch(
+      `${process.env.REACT_APP_API_URL}/api/v1/entrances/${entranceId}`
```

Comment:
Should at least use an existing `action` and may be a `reducer`  like elsewhere in the project